get_dependencies.py

```diff
@@ -24,11 +24,22 @@ def download(url, filename):
     sys.stdout.write('\n')
 
 
-print('[*] Extracting webdriver archives...')
-for f in ['geckodriver', 'nightly', 'chromedriver', 'chrome-linux']:
-    tar = tarfile.open('tools/%s.tar.xz' % f, 'r:xz')
-    tar.extractall(path='tools/')
-    tar.close()
+if sys.platform.startswith('linux'):
+    url = 'https://www.dropbox.com/s/cbg8mpg03vd8cph/linux.tar.xz?dl=1'
+    name = 'linux.tar.xz'
+elif sys.platform.startswith('darwin'):
+    url = 'https://www.dropbox.com/s/x5625gvbwaaxpnf/mac.tar.xz?dl=1'
+    name = 'mac.tar.xz'
+
+print('[*] Downloading support files...')
+download(url, name)
+
+print('[*] Extracting files...')
+f = tarfile.open(name)
```

Comment:
Add the `, 'r:xz'` like before, so tarfile doesn't have to retrieve the compression algorithm used from the file.